### PR TITLE
header: add navigation on mobile

### PR DESCRIPTION
**Pull request by augusto-jm-amaral**
## Description
Add header navigation on mobile.

image

Closes https://github.com/pagarme/mercurio/issues/194

## Your checklist for this pull request

:rotating_light: Please review this items for a good pull request. :four_leaf_clover:

1. I've read the project's [Contributing Guidelines](CONTRIBUTING.md)
1. My commits are well written and follow [pagarme/git-style-guide](https://github.com/pagarme/git-style-guide)
1. My changes are well covered by **tests** and **logs**
1. I've updated the project docs (if needed)
1. I fell safe about this implementation

In a good pull request, everything above is true :relaxed:

### CODE REVIEW

File: src/containers/Checkout/index.js [hunk 1]
```diff
@@ -448,6 +448,12 @@ class Checkout extends Component {
             logoSrc={logo}
             steps={pages}
             activeStep={machineState.value}
+            handleCloseButton={this.close}
+            handlePreviousButton={
+              pages[0].page !== machineState.value ?
```
Comment (thiagommedeiros) [inline src/containers/Checkout/index.js]: adiciona isso numa const com um nome bacanudo pra ficar mais facil o entendimento =D

Comment (augusto-jm-amaral) [reply src/containers/Checkout/index.js]: Boa, vou fazer

File: src/containers/Checkout/index.js [hunk 2]
```diff
@@ -414,6 +414,10 @@ class Checkout extends Component {
       .filter(value => !hasRequiredPageData(value.page, this.props))
       .map(makeStepInvisible)
 
+    const previousButtonHeader = pages[0].page !== machineState.value ?
```
Comment (thiagommedeiros) [inline src/containers/Checkout/index.js]: @augusto-jm-amaral eu me referia a este trecho `pages[0].page !== machineState.value`

adicionar isso numa const, algo como:

```js
const isNotFirstPage = pages[0].page !== machineState.value 
```

Comment (augusto-jm-amaral) [reply src/containers/Checkout/index.js]: KKkk eita, vou ajustar perae!

**Pull request merged**